src/cudamatrix/cu-array.h

```diff
@@ -87,6 +87,7 @@ class CuArray {
   /// is done via memcpy.  So be very careful calling this function if your
   /// objects are more than plain structs.
   void CopyToVec(std::vector<T> *dst) const;
+  void CopyToVec(T *dst) const;
```

Comment:
Please add a comment, e.g.
/// Version of the above that copies contents to a host array.
/// 'dst' must already be correctly sized.